### PR TITLE
Change more symbols to letter-like

### PR DESCRIPTION
**Pull request by TiagoCavalcante**

### CODE REVIEW

**Comment by mmatera** [comment]
LGTM Thanks for the work!

**Comment by TiagoCavalcante** [comment]
I converted it to draft again because there are more symbols that should be letter-like, but doing it fail the tests.  
mathics-core tests needs to be updated, some of they are testing against the ASCII version of the symbol, which sometimes is only available in Unicode. (e.g.: E vs ⅇ).

**Comment by rocky** [comment]
> I converted it to draft again because there are more symbols that should be letter-like, but doing it fail the tests.
> mathics-core tests need to be updated, some of them are testing against the ASCII version of the symbol, which sometimes is only available in Unicode. (e.g.: E vs ⅇ)

Yes, I think what is needed in mathics-core is to check if the result is unicode and compare against a unicode version if that exists.

**Comment by TiagoCavalcante** [comment]
I think it is good now, I didn't changed to ASCII vs Unicode thing, but 147 more symbols are letter-like now.

File: mathics_scanner/data/named-characters.yml [hunk 1]
```diff
@@ -1597,6 +1597,7 @@ DifferentialD:
   amslatex: "\\DifferentialD"
   esc-alias: dd
   has-unicode-inverse: true
+  # This can't be letter-like bacause it is used in derivatives as a function
```
Comment (rocky) [inline mathics_scanner/data/named-characters.yml:1600]: bacause -> because

**Review by rocky — APPROVED** [review]
LGTM - see small typo. 

When this doesn't cause a problem in mathics core I am fine with merging and iterating on it more if it needs that. 

Thanks for doing this. I realize this is a huge, onerous task. 

Hopefully we can update [readthedocs information](https://mathics-development-guide.readthedocs.io/en/latest/translation-tables/wl-to-unicode-conversion.html) at some point.

File: mathics_scanner/data/named-characters.yml (hunk 1 above)
Comment (TiagoCavalcante) [reply mathics_scanner/data/named-characters.yml]: Fixed now.
